docs/sources/getting-started/glossary.md

```diff
@@ -34,6 +34,12 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
       An exemplar is any data that serves as a detailed example of one of the observations aggregated into a metric. An exemplar contains the observed value together with an optional timestamp and arbitrary labels, which are typically used to reference a trace.
     </td>
   </tr>
+  <tr>
+    <td style="vertical-align: top">Explore</td>
+    <td>
+      Explore allows a user to focus on a building a query. User can refine the query to return the expected metrics before building a dashboard.
```

Comment:
Link to the Explore topic.

Reply:
Unable to add links to Glossary at this time.